docs/examples/expressions.ipynb

```diff
@@ -0,0 +1,170 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# DALI expressions and arithmetic operators\n",
+    "\n",
+    "In this example, we will see how to use arithmetic operators in DALI Pipeline."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import types\n",
+    "import collections\n",
+    "import numpy as np\n",
+    "from nvidia.dali.pipeline import Pipeline\n",
+    "import nvidia.dali.ops as ops            \n",
+    "import nvidia.dali.types as types\n",
+    "\n",
+    "batch_size = 1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Defining the iterator\n",
+    "\n",
+    "We will use custom iterator producing small tensors filled with series of numbers, so we can easily inspect the results."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class ExternalInputIterator(object):\n",
+    "    def __init__(self, batch_size, left_type, right_type):\n",
+    "        self.batch_size = batch_size\n",
+    "        self.left_type = left_type\n",
+    "        self.right_type = right_type\n",
+    "\n",
+    "    def __iter__(self):\n",
+    "        self.i = 0\n",
+    "        self.n = 128\n",
+    "        return self\n",
+    "\n",
+    "    def __next__(self):\n",
+    "        left = []\n",
+    "        right = []\n",
+    "        for sample in range(self.batch_size):\n",
+    "            left.append(np.array([sample + self.i], dtype = self.left_type))\n",
```

Comment:
Can we have this kind of tests also in python tests?

Reply:
I added one.